### ISSUE TITLE
Add --quiet option to `authelia crypto rand` command

### ISSUE THREAD

**Issue by jonasgeiler** [authelia/authelia#7467]
### Description

When running `authelia crypto rand --length 64`, the following is printed to the terminal:
```
Random Value: 2RUT9NzHzpXUp1fMDC2QaNnLk5ddYfoFwqjofwKhF0YFtaFnhu09eHkPzU6KsDHC
```

If I wanted to pipe this output into a file like `authelia crypto rand --length 64 > /data/secrets/jwt_secret`, the file will contain something like this:
```
Random Value: jn81Q2TardAO4EPcgG4mfAmvjtIDE8SYaZ3QgI9LjA8ZGSwMeLIMC61d08wKs4vn
```
As you can see the `Random Value: ` part is also written to the file.

It would be nice if I could provide a `--quiet` option like `authelia crypto rand --length 64 --quiet > /data/secrets/jwt_secret`, so the file will only contain this:
```
jn81Q2TardAO4EPcgG4mfAmvjtIDE8SYaZ3QgI9LjA8ZGSwMeLIMC61d08wKs4vn
```

Obviously you can remove the `Random Value: ` part yourself with a little bit of Bash scripting, but it would still be nice to have this option for simplicity and compatibility with any changes to the output in the future.

### Use Case

The main use case is using the `authelia crypto rand` command in Bash scripts and the-like. You could put the output into a Bash variable or pipe it into a file for use with Docker secrets and such. Or maybe someone wants just the random string without other stuff for easier copy-pasting?

### Details

The `authelia crypto rand` command should have a `--quiet` option which removes and redundant output from the command. Only the actual random string should be outputted.

### Pre-Submission Checklist

- [X] I agree to follow the [Code of Conduct](http://www.authelia.com/code-of-conduct)

- [X] I have checked for related issues and checked the documentation

**Comment by jonasgeiler** [comment]
> However, you can achieve the desired result using standard shell tools to filter the output. For example, you can use `awk` to extract just the random string:
> 
> ```shell
> authelia crypto rand --length 64 | awk '{print $3}' > /data/secrets/jwt_secret
> ```

Of course this would be possible, but as I said the output might change in the future and could break your script, for example if the output changes from `Random Value: abc` to `Random Alphanumeric Value: abc` or something like that.

**Comment by jonasgeiler** [comment]
I'd be ready to work on a PR myself to add this feature. Let me know if I can!

**Comment by james-d-elliott** [comment]
This sounds like a decent idea. Writing to a file instead is more consistent with other areas of the CLI however, so since it also matches the intention that'd be the direction I'd go.

**Comment by jonasgeiler** [comment]
> @james-d-elliott wrote:
> This sounds like a decent idea. Writing to a file instead is more consistent with other areas of the CLI however, so since it also matches the intention that'd be the direction I'd go.

Yeah that's true - the `authelia crypto certificate` command and such all output to a file, so for consistency it's probably better this way.
Are you gonna work on this? As I said, I'd be ready for a PR myself.

**Comment by james-d-elliott** [comment]
I'm in the middle of doing the passkey and OIDC features but will lump this in with release finalization if you don't get around to it. Would welcome the PR.

**Comment by jonasgeiler** [comment]
> @james-d-elliott wrote:
> Would welcome the PR.

Oh apparently @Crowley723 beat me to it. Just wanted to look into this again right now 😅 
Thanks, @Crowley723!

**Comment by Crowley723** [comment]
> > @james-d-elliott wrote:
> > Would welcome the PR.
> 
> Oh apparently @Crowley723 beat me to it. Just wanted to look into this again right now 😅 
> Thanks, @Crowley723!

Glad I could help!

**Comment by james-d-elliott** [comment]
Closed by d386d06c2e0b0d1e20dbc587dc32ee4da1b5f825
